assets/constants.js

```diff
@@ -528,6 +528,8 @@ var constants = {
 
   "NUM_QUESTIONS_PER_PAGE": 10,
 
+  "NUM_QUESTION_SKILL_LINKS_PER_PAGE": 30,
```

Comment:
I think it might help to have a comment saying which page this refers to.

Reply:
Replaced it with MAX_SKILLS_PER_QUESTION